### PR TITLE
Feature/player selection cancel

### PR DESCRIPTION
**Pull request by m4rtint**
- This gives players the ability to press cancel

### CODE REVIEW

**Comment by m4rtint** [comment]
[Player Selection - Allow Cancel](https://trello.com/c/4rSfzMry/467-player-selection-allow-cancel)

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/PlayerSelectionViewController.cs [hunk 1]
```diff
@@ -181,6 +202,28 @@ private void OnEnable()
...
         }
+
+        private void ChangeCarSelectionState(CarSelectionState state)
+        {
+            StartCoroutine(SetChooseCarState(state));
+        }
+        
+        private IEnumerator SetChooseCarState(CarSelectionState state)
+        {
+            var timeToWait = m_disableButtonsTimeInTransition;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/PlayerSelectionViewController.cs:213]: get rid of var

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs [hunk 2]
```diff
@@ -84,7 +83,10 @@ public void FadeOut()
         {
             //Show "Confirmed" button
             SetCarConfirmed(true);
-            CommonAnimations.FadeOutAlpha(m_canvasGroup, m_fadeTime).SetDelay(m_delayDuration);
+            CommonAnimations.FadeOutAlpha(m_canvasGroup, m_fadeTime).SetDelay(m_delayDuration).OnComplete(() =>
+            {
+                gameObject.SetActive(false);
+            });;
```
Comment (DJSteins) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs:89]: Fix double semi-colon

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/PlayerSelectionViewController.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/PlayerSelectionViewController.cs]: Changed

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs (hunk 2 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/CarStats/CarStatSelectionViewController.cs:89]: Removed

**Pull request merged**